daisy_workflows/image_import/windows/translate.ps1

```diff
@@ -356,10 +361,6 @@ try {
   else {
     # Since 32-bit GooGet packages are not provided via repository, the only option is to install them from a local source.
     Install-32bitPackages
```

Comment:
there seem to be 2 dimensions, 32vs64? and Windows10 vs !Windows10; did you mean to make the function never run on 32bit as well as not running on Windows10?

Reply:
Gosh, thanks for spotting this, I've been looking at this for too long today! My intention was to bypass `Configure-Power` for all versions of Windows 10, which will unblock the user that we're working with. In parallel to unblocking them, I'm working to see if we can create a better filter than just "Windows 10", since we're seeing that some versions of Windows 10 work here, while others fail.